.circleci/config.yml

```diff
@@ -61,7 +61,7 @@ jobs:
   # deps has to be run before the other jobs and basically run "npm install" and cache it
   deps:
     docker: &node_image
-      - image: cimg/node:18.13
+      - image: cimg/node:21.6.1
```

Comment:
It is not only "node_browsers_image". 
Shouldn't we stick with the 20.11 "LTS" version for now?
It has impact on my other building / deployment tools as well.